lib/index.js

```diff
@@ -74,17 +74,10 @@ export default {
 
         // Call ERB to "de-templatize" the code
         return helpers.exec(this.erbPath, erbArgs,
-          { stdin: text, cwd: fileDir }
+          { stdin: text.replace(/<%=/g, '<%'), cwd: fileDir }
```

Comment:
I already looked at the original commit where that extra scope was introduced and it hints at rails ERB being marked as `text.html.ruby`, but I don't think that's the case anymore. The original code would specifically look for `text.html.ruby` and only "descramble" that specific scope, but it must have been changed along the way to do the opposite. I'll investigate further.